### PR TITLE
Idva5 493 limited what is your role

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/cy/what-is-your-role.json [hunk 1]
```diff
@@ -2,8 +2,10 @@
     "title": "What is your role? welsh",
     "soleTrader":"I am the sole trader welsh",
     "someoneElse":"I am someone else welsh",
-    "member": "I am a member of welsh ",
-    "memberOfGoverningBody": "I am a member of the governing body of welsh ",
+    "member": "welsh I am a member of ",
+    "memberOfGoverningBody": "welsh I am a member of the governing body of ",
+    "director": "welsh I am a director of ",
+    "generalPartner": "welsh I am a general partner of ",
```
Comment (swalke-ch) [inline locales/cy/what-is-your-role.json:8]: Can we follow same pattern of adding 'welsh' at the end of the line please? just to avoid confusion

Comment (sardhani-ch) [reply locales/cy/what-is-your-role.json:8]: If I add in the end, welsh will appear in between the label like  "I am a director of welsh MORRIS Accounting". I felt its good to be either at the beginning or end of label and not in between.

Comment (swalke-ch) [reply locales/cy/what-is-your-role.json:8]: okay, that should be fine then

**Review by swalke-ch — APPROVED** [review]

**Pull request merged**